pkgs/development/python-modules/python-pidfile/default.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  psutil,
+}:
+
+buildPythonPackage rec {
+  pname = "python-pidfile";
+  version = "3.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-HhCX30G8dfV0WZ/++J6LIO/xvfyRkdPtJkzC2ulUKdA=";
+  };
+
+  propagatedBuildInputs = [
+    psutil
+  ];
+
```

Comment:
If there are no test then `doCheck = false;` should be added. 

Let's see...https://github.com/mosquito/python-pidfile/issues/7

Reply:
Thanks for the feedback!